components/AppHeader.vue

```diff
@@ -157,14 +157,21 @@ import logTrackerEvent, { setLoggerUser } from '~/utils/logger'
 import { IS_TESTNET } from '~/constant'
 
 const walletModule = namespace('wallet')
+const bookApiModule = namespace('book-api')
 
 @Component
 export default class AppHeader extends Vue {
   @walletModule.Getter('getType') walletType!: string | null
   @walletModule.Action('disconnectWallet') disconnectWallet!: () => void
   @walletModule.Action('openConnectWalletModal') openConnectWalletModal!: (params: { language: string, fullPath?: string }) => Promise<any>
   @walletModule.Action('initWallet') initWallet!: (params: { method: any, accounts: any, offlineSigner?: any }) => Promise<any>
+  @walletModule.Action('signMessageMemo') signMessageMemo!: (action: string, permissions?: string[]) => Promise<any>
   @walletModule.Getter('getWalletAddress') currentAddress!: string
```

Comment:
Please check where do we use `getWalletAddress `, and consider replacing them all with `sessionWallet`

Reply:
[🚸 Improve login UX flow](https://github.com/likecoin/app-like-co/pull/474/commits/e6861c5e5254346b34d2cc1adef7a12f1f2ae05c)